src/constraint/spec.ts

```diff
@@ -534,30 +534,6 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
       }
       return true;
     }
-  },
-  {
-    name: 'scaleZeroMustMatchScaleType',
```

Comment:
A correct commit message would be "remove `scaleZeroMustMatchScaleType` constraint and its test."